component-library/component-lib/src/lib/form-components/input/input.component.ts

```diff
@@ -112,6 +115,8 @@ export class InputComponent
   @Input() errorMessages?: IErrorPairs[];
   @Output() focusEvent = new EventEmitter();
 
+  @ViewChild('inputEl') inputEl?: ElementRef;
```

Comment:
Looks like you've encapsulated it correctly using Renderer2, but for sanity sake, check with Alexandre as well.